php/Terminus/User.php

```diff
@@ -73,12 +75,21 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
-  public function getId() {
-    return $this->id;
+  public function instruments() {
+    $instruments       = new Instruments(array('user' => $this));
```

Comment:
That's because we shouldn't have `fetch` in here at all. If this is purely an accessor, it should just return the property. And in the constructor, it should just initialize the property. Invocation of a fetch should be done out side of the Model.